lib/helpers/delta-grid.js

```diff
@@ -0,0 +1,81 @@
+module.exports = function (Handlebars, _) {
+  Handlebars.registerHelper('deltaGrid', function () {
```

Comment:
Is there a reason this is a helper rather than a partial? I'm seeing mostly HTML string concatenation in here. If it's just to avoid having a template loaded as a file, you can always use the ol' `Function.prototype.toString`